packages/terra-date-picker/CHANGELOG.md

```diff
@@ -11,7 +11,11 @@ Unreleased
 
 ### Changed
 * Update tests for terra-toolkit v5 and terra-dev-site v5 changes
+<<<<<<< HEAD
```

Comment:
remove merge conflict markers

Reply:
sure you don't want these in here?? 😄

Reply:
fixed: https://github.com/cerner/terra-framework/pull/729/commits/0b7264816a20a234bae654707f72b1c8f1f16230